index.html

```diff
@@ -14,37 +14,45 @@ <h2>Mark</h2>
         <nav>
             <img src="images/Enabled.png" alt="" class="image-icon">
         </nav>
+
+        <nav>
+            <ul class="nav-content">
+                <li><a>Portfolio</a></li>
+                <li><a>About</a></li>
+                <li><a>Contact</a></li>
+              </ul>
+        </nav>
```

Comment:
- [ ] Kindly, as mentioned by the previous reviewer you should match the Figma design as much as possible. That includes margin, padding, color, images, & so on. Currently, your logo is missing proper margins/paddings.

## Your design
![image](https://github.com/MarkYegon7/Portfolio-Setup/assets/86069740/6ef65a32-c36e-495c-b0d0-9da2dc832035)

## Figma
![image](https://github.com/MarkYegon7/Portfolio-Setup/assets/86069740/8b8f3bd7-cc5b-447a-a2f6-a372dfde02ef)

Reply:
Fixed the problem.